MiniApp/Classes/MiniApp.swift

```diff
@@ -15,11 +15,11 @@ public class MiniApp: NSObject {
     /// Create a Mini App for the given appId. Mini app will be downloaded and cached in local.
     ///
     /// - Parameters:
-    ///   - appId: Unique Application Id of Mini App
+    ///   - appInfo: Mini App info object
     ///   - completionBlock: A block to be called on successful creation of [MiniAppView] or throws errors if any. Completion blocks receives the following parameters
-    ///         -   [MiniAppView]: Mini App View placeholder that hosts the Mini App
+    ///         -   MiniAppDisplayProtocol: Protocol that helps the hosting application to communicate with the SDK
```

Comment:
I think a bit more specific description would be nice. e.g. protocol for <specific purpose>